README.md

```diff
@@ -109,11 +109,12 @@ Example: Having a slug like `/products/a`, renaming it to `/products/b` then ren
 
 ### Points of improvements 
 
-The same mechanism could be made for taxon slugs, and obviously for any other resource that has a slug. Feel free to submit a PR.
+The same mechanism could be made other resources that has a slug. Feel free to submit a PR.
```

Comment:
For now we can just remove this whole section because there are no other core resources that has slugs I guess?

Reply:
True, I was thinking to look into the BitBag CMS, but that probably should be a stand alone repository.